src/test/java/com/bullhornsdk/data/api/helper/RestJsonConverterTest.java

```diff
@@ -154,4 +159,28 @@ public void testStandardFileContentWithExtraProp() {
         assertEquals("SomeContent", file.getFileContent());
         assertEquals("FileName", file.getName());
     }
+
```

Comment:
Can we create another Test case that ensures that if the OneToMany relationship is completely null, that the JSONObject does not get populated with an empty replaceAll?

Reply:
If the OneToMany relationship is null, it will be ignored by Jackson and won't be present in the resulting JSON.